<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:blue;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
</string>
   </property>
   <property name="text">
    <string>Primer Orden</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Back_P1_Pulso">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>51</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
</string>
   </property>
   <property name="text">
    <string>Frecuencia en Hertz del polo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>150</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
</string>
   </property>
   <property name="text">
    <string>Frecuencia en Hertz del cero:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="P1_Continuar_Pulso">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>540</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
   <property name="text">
    <string>Graficar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="P1_P_Pulso">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="P1_C_Pulso">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>220</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
</string>
   </property>
   <property name="text">
    <string>Amplitud del Pulso u(t):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="P1_A_Pulso">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="P1_G_Pulso">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>370</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>300</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
</string>
   </property>
   <property name="text">
    <string>Ganancia Pasante o Maxima:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>440</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
</string>
   </property>
   <property name="text">
    <string>Seleccionar Filtro:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="P1_Filtro">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>510</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
   <item>
    <property name="text">
     <string>Pasa Bajos</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pasa Altos</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pasa Todo</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
